### DISCUSSION TITLE
No keyboard input when building to executable

### DISCUSSION THREAD

**Discussion by entangledothers** [cables-gl/cables_docs#283] [! Welcome - readme first!]
I've been building an experience with the Physics Room template, which in-editor and in-browser works perfectly. However when building as an executable (tested on win & mac) only the mouse-look and mouse-click events work*, no keyboard input seems to register. 

Does anyone have an idea of why electron doesn't seem to be handling any keyboard input? The experience itself renders fine and I've tested both with the default template with no edits as well as the customised scene I've been building.

This has been tested on windows 10 as well as M1 mac 12.4 (running intel build in rosetta mode). 

`*` _so for example the mobile overlay for locomotion in the default physics room scene can be manipulated via the mouse cursor, but the WASD keys do not work._

**Comment by steam0r** [comment]
hey, i changed some things how electron handles keyboard input and this fixed the room template for me. sadly you will have to export this to a different (new) repository to get the fix, but please give it a try!

we are thinking of moving this away from github (to also be able to solve the m1/arm issue that you posted)

**Reply by entangledothers** [reply]
That seems to have done it! Exported to a new repo and now keyboard input works where it previously didn't! If you move elsewhere would it be something you host yourselves or would it be more like a local build process setup? (both options sound fine to me at least).

Thanks very much for the fix!

**Reply by steam0r** [reply]
it would be the same process just on our servers, this would then maybe give us the ability to sign these binaries and remove this rather tedious github integration thingie, that also seems errorprone and sometimes plain unavailable. thanks for testing again!
